cxotime/tests/test_cxotime.py

```diff
@@ -207,3 +207,29 @@ def test_strict_parsing():
 
     with pytest.raises(ValueError, match='Input values did not match the format class greta'):
         CxoTime('2000001.123', format='greta')
+
+
+@pytest.mark.parametrize(
+    'date',
+    ['2001:002',
+     '2001:002:03:04',
+     '2001:002:03:04:05',
+     '2001:002:03:04:05.678'])
+def test_date2secs(date):
```

Comment:
There might be some value to cross validating here to Chandra.Time.date2secs, but I think this is good.